Fase (#523)
SteamDatabase/FileDetectionRuleSets: descriptions/Engine.Fase.md

> [**FASE**](https://github.com/DSkywalk/fase/tree/master) stands for "Fucking Awesome Spectrum Engine".
> It is an engine for sprites, tiles, and maps, made to interface with ZX Basic, C and ASM.

Reviewer: ```suggestion
[**Fucking Awesome Spectrum Engine (FASE)**](https://github.com/DSkywalk/fase) is a small open source engine.
```

Author: Ok mx minimalist.
